code/game/objects/items/weapons/clown_items.dm

```diff
@@ -68,6 +68,13 @@
 	var/honk_sound = 'sound/items/bikehorn.ogg'
 	var/cooldowntime = 20
 
+/obj/item/weapon/bikehorn/syndie
+	desc = "Guaranteed to make you the subject of a Doctor Doctor joke."
+	attack_verb = list("VIOLENTLY HONKED", "VICIOUSLY MOCKED")
+	origin_tech = "combat=3;syndicate=1"
+	force = 25
+	no_embed = TRUE

```

Comment:
`no_embed` is an unused var.

The ability to embed object by attacking someone with them was purged years ago.